src/module.ts

```diff
@@ -231,13 +232,12 @@ export default defineNuxtModule<ModuleOptions>({
       const { withTrailingSlash, withoutTrailingSlash } = await import('ufo')
       const routerPrefix = isNuxt3() ? route : undefined
       const _viewerDevMiddleware = createServer({ tailwindConfigProvider: () => tailwindConfig, routerPrefix }).asMiddleware()
-      const viewerDevMiddleware = (req, res) => {
-        if (req.originalUrl === withoutTrailingSlash(route)) {
-          res.writeHead(301, { Location: withTrailingSlash(req.originalUrl) })
-          return res.end()
+      const viewerDevMiddleware = eventHandler((event) => {
+        if (event.req.url === withoutTrailingSlash(route)) {
+          return sendRedirect(event, withTrailingSlash(event.req.url), 301)
         }
-        _viewerDevMiddleware(req, res)
-      }
+        _viewerDevMiddleware(event.req, event.res)
```

Comment:
We could convert `_viewerDevMiddleware` to an event handler using `fromNodeMiddleware(_viewerDevMiddleware)` and await on it

Reply:
Shall I wait before merging then?

Reply:
I think we could merge as is and possibly improve later. When I implement the suggestion, I get a 404 on `/_tailwind/config.json`. cc: @pi0

Reply:
@Atinux Feel free to merge if it is working as is now. will check afterwards for compatibility...